lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -71,7 +71,7 @@ public static software.amazon.awssdk.core.retry.RetryMode getRetryMode(RetryMode
             return switch (retryMode) {
                 case STANDARD -> software.amazon.awssdk.core.retry.RetryMode.STANDARD;
                 case LEGACY -> software.amazon.awssdk.core.retry.RetryMode.LEGACY;
-                case ADAPTIVE -> software.amazon.awssdk.core.retry.RetryMode.ADAPTIVE;
+                case ADAPTIVE -> software.amazon.awssdk.core.retry.RetryMode.ADAPTIVE_V2;
```

Comment:
please update file-system-s3 doc

Reply:
What should I write there except for the fact that ADAPTIVE_V2 is no longer experimental?